templates/includes/sidebar.html

```diff
@@ -1,4 +1,20 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      <a href="{% url 'taxi:driver-detail' user.id %}">Driver: {{ user.first_name }}</a>
```

Comment:
Ensure that the URL pattern name `'taxi:driver-detail'` is correctly defined in your `urls.py` with the expected parameter. If `user.id` is not the correct parameter, this link may not work as intended.